Aventra-MyEID-PKI-card.md

```diff
@@ -10,41 +10,58 @@ Optional features include magnetic stripe, signature panel, holograms, security
 
 The cards can be personalized both visually and electrically by Aventra according to customer specifications, or the customers can personalize the cards themselves using ActivePerso Manager developed by Aventra, or software from other parties.
 
-MyEID is certified by Microsoft and supports Smart Card Plug and Play.
+Aventra participates in development and testing of OpenSC, keeping MyEID support up to date with new MyEID versions. 
 
-> MyEID version 4 has been released, adding support for Elliptic Curve Cryptography and many other new features.
+> MyEID is certified by Microsoft and supports Smart Card Plug and Play.
+
+> MyEID 5.0 will be released soon. It will be eIDAS and Common Criteria EAL4+ certified.  
```

Comment:
I think it would be good to state only things that we know, rather than things that we expect to happen. In the past, this wiki has not received many updates on things like this.

Reply:
Ok, I will rephrase this also.